### PR TITLE
Replace "MaxCDN" with "StackPath".

### PR DESCRIPTION
**Pull request by XhmikosR**
Only thing left is the GitHub organization and the Travis notification email.

### CODE REVIEW

**Comment by XhmikosR** [comment]
```
C:\Users\XhmikosR\Desktop>curl -ILl https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js
HTTP/2 200
accept-ranges: bytes
access-control-allow-origin: *
cache-control: max-age=31104000
content-length: 48944
content-type: application/javascript; charset=utf-8
date: Tue, 27 Mar 2018 17:56:35 GMT
etag: "14d449eb8876fa55e1ef3c2cc52b0c17"
expires: Fri, 22 Mar 2019 17:56:35 GMT
last-modified: Tue, 20 Feb 2018 05:58:03 GMT
server: NetDNA-cache/2.2
vary: Accept-Encoding
x-cache: HIT
x-hello-human: Say hello back! @getBootstrapCDN on Twitter

C:\Users\XhmikosR\Desktop>curl -ILl https://stackpath.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js
HTTP/1.1 200 OK
Accept-Ranges: bytes
Access-Control-Allow-Origin: *
Cache-Control: max-age=31100565
Connection: Keep-Alive
Content-Length: 48944
Content-Type: application/javascript; charset=utf-8
Date: Tue, 27 Mar 2018 17:56:37 GMT
ETag: 1519106283
Last-Modified: Tue, 20 Feb 2018 05:58:03 GMT
X-Hello-Human: 'Say hello back! @getBootstrapCDN on Twitter'
X-HW: 1522173397.dop008.lo4.t,1522173397.cds043.lo4.shn,1522173397.dop008.lo4.t,1522173397.cds030.lo4.c
```

1. http/2
2. why the difference in cache control header
3. `server`, if it's going to stay like that, we need to fix our tests
4. `x-cache`, the same as above
5. the `x-hello-human` header has single quotes around it

File: tests/functional_test.js [hunk 1]
```diff
@@ -13,32 +13,20 @@ const helpers   = require('./test_helper.js');
...
-    'x-cache': undefined,
-
-    'accept-ranges': undefined,
-    'access-control-allow-origin': '*',
-
-    // the following are set as undefined because www
-    // and assets (js/css) differ
-    'server': undefined,
-    'x-hello-human': undefined,
-    'cache-control': undefined
```
Comment (jmervine) [inline tests/functional_test.js]: I prefer these spaced out the way they were for readability, but I won't block on it.

**Comment by XhmikosR** [comment]
Let's wait until tests are green :)

I have disabled the `x-cache` and `server` headers checks; we should decide what to with them.

PS. I think we are still hitting the timeout issue...

**Comment by XhmikosR** [comment]
All right, so this PR as is, we just hit the timeout issue only. Should we merge?

**Comment by XhmikosR** [comment]
There's a blocking issue with gzip not being enabled so we need to wait until this is fixed.

Apart from that, we should decide if we will leave `server` to say Amazon S3, replace it with something else, or just hide it.

File: package.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
     "puglint": "pug-lint .",
     "lint": "npm run eslint && npm run puglint",
     "mocha": "mocha --timeout 15000 --slow 500 --exit -i -g \"functional\" -R dot ./tests/",
-    "mocha:suite": "mocha --timeout 15000 --slow 500 --exit -R dot ./tests/",
+    "mocha:suite": "mocha --timeout 15000 --slow 500 --exit ./tests/",
```
Comment (jmervine) [inline package.json]: We should probably revert this line before releasing. It makes the output insanely long. Another option might be `-R tap`, which is still verbose, but should be about 1/3 the output.

**Comment by jdorfman** [comment]
WTF is happening? 

image

**Comment by XhmikosR** [comment]
The timeouts as usual... I'm not sure if it's the CDN's fault after all. But I don't have any solid proof for anything. I mean, locally it works, but I'm on Windows, so we still don't have the ability to run the same number of tests.

**Comment by jmervine** [comment]
Limiting the number of functional tests run in parallel with `async.each`, I have a theory that this might be related to a low limit of concurrent http connections to prevent abuse in linux based environments.

**Comment by jmervine** [comment]
With tests now passing on Heroku CI, I'd like to drop AppVeyor and Travis.

**Comment by jmervine** [comment]
To @jdorfman's point, until Heroku CI supports public results, we can't really switch to it. I still vote to disable AppVeyor, I don't think we need two CI's running.

**Comment by XhmikosR** [comment]
OK, I believe we are finally ready 🎉 

1. CIs are green
2. gzip is fixed
3. `cache-control` is set again to `max-age=31536000`
4. `server` header is hidden
5. `X-Hello-Human` is being set
6. `Access-Control-Allow-Origin` is being set
7. Headers' case is consistent

```
C:\Users\XhmikosR\Desktop>curl -ILl -H "Accept-Encoding: gzip" https://stackpath.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js
HTTP/1.1 200 OK
Date: Wed, 04 Apr 2018 07:44:09 GMT
Connection: Keep-Alive
Accept-Ranges: bytes
ETag: 1519106283
Cache-Control: max-age=31536000
Content-Encoding: gzip
Content-Length: 13105
Content-Type: application/javascript; charset=utf-8
Last-Modified: Tue, 20 Feb 2018 05:58:03 GMT
X-Hello-Human: Say hello back! @getBootstrapCDN on Twitter
Access-Control-Allow-Origin: *
Vary: Accept-Encoding
X-Cache: HIT

C:\Users\XhmikosR\Desktop>curl -ILl -H "Accept-Encoding: gzip" https://stackpath.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css
HTTP/1.1 200 OK
Date: Wed, 04 Apr 2018 07:44:11 GMT
Connection: Keep-Alive
Accept-Ranges: bytes
ETag: 1519106276
Cache-Control: max-age=31536000
Content-Encoding: gzip
Content-Length: 20563
Content-Type: text/css; charset=utf-8
Last-Modified: Tue, 20 Feb 2018 05:57:56 GMT
X-Hello-Human: Say hello back! @getBootstrapCDN on Twitter
Access-Control-Allow-Origin: *
Vary: Accept-Encoding
X-Cache: HIT
```

So, I think we are finally ready.

**Comment by XhmikosR** [comment]
Also, note that tests are significantly faster.

Travis CI develop vs this branch: **4083 passing (5m)** vs **3823 passing (1m)**

**Comment by jdorfman** [comment]
LGTM, merge when ready

**Pull request merged**